### PR TITLE
[RM CH04] Fix 4.2.5 Profile Capability Mapping

### PR DESCRIPTION
**Pull request by ASawwaf**
https://github.com/cntt-n/CNTT/issues/1748#event-3506937302
Fix 4.2.5 Profile Capability Mapping
SmartNIC
FPGA
GPU
Allocation ratio

### CODE REVIEW

**Comment by rabi-abdel** [comment]
@walterkozlowski @TFredberg can you please approve based on the discussion we had on the @cntt-n/ref-edge meeting?

File: doc/ref_model/chapters/chapter04.md [hunk 1]
```diff
@@ -475,12 +475,12 @@ n100, n200, n300, n400, n500, n600 | N | Y | N
 | `e.cap.011`<br />(Programmable Acceleration)                | No                       | No                       |                                                                     |
 | `e.cap.012`<br />(Enhanced Cache Management)                | E                        | E                        |                                                                     |
 | `e.cap.013`<br />(SR-IOV over PCI-PT)                                 | Yes                      | No                       | |
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter04.md:477]: @ASawwaf needs to flip
```suggestion
| `e.cap.013`<br />(SR-IOV over PCI-PT)                                 | No                      | Yes                       | |
```

File: doc/ref_model/chapters/chapter04.md [hunk 2]
```diff
@@ -475,12 +475,12 @@ n100, n200, n300, n400, n500, n600 | N | Y | N
 | `e.cap.011`<br />(Programmable Acceleration)                | No                       | No                       |                                                                     |
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter04.md:475]: @ASawwaf 
```suggestion
| `e.cap.011`<br />(Programmable Acceleration)                | No                       | Yes/No                       |                                                                     |
```

**Comment by rabi-abdel** [comment]
@karinesevilla I believe your input was addressed, can you please re-review and incase it was not, propose a commit suggestion?

**Comment by rabi-abdel** [comment]
merging after 48 hours of addressing feedback from everyone

**Pull request merged**